libs/core/mappers/anime.mapper.ts

```diff
@@ -0,0 +1,56 @@
+
+import { Injectable } from '@angular/core';
+
+import { Anime } from '../models/anime';
+import { AnimeDto } from '../dtos/anime.dto';
+import { typeMappingFromDto } from '../records/type-from-dto';
+import { statusMappingFromDto } from '../records/status-from-dto';
+import { typeMappingToDto } from '../records/type-to-dto';
+import { statusMappingToDto } from '../records/status-to-dto';
+
+import { Mapper } from './mapper';
+
+/** Anime mapper map Model to Dto and vice vera Dto to Model. */
```

Comment:
It can also map other things if necessary
```suggestion
/** Anime mapper. */
```